qstash/features/queues.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Queues"
+---
+
+The queue concept in Qstash allows ordered delivery(FIFO) and also controlled parallelism. 
+Here we list common use cases for Queue and how to use them.
+See the [API doc](../api/queues/upsert) for the full list of related Rest APIs.
+
+## Ordered Delivery
+
+With Queues, the ordered delivery is guaranteed by default. 
+This means:
+
+- Your messages will be queued and sent one by one in FIFO order.
+- The next message will wait for retries of the current one if it can not be delivered because your endpoint returns non-2xx code. 
+- Next message will wait for [callbacks](callbacks#what-is-a-callback) or [failure callbacks](callbacks#what-is-a-failure-callback) to finish.
+
+```bash
+curl -XPOST -H 'Authorization: Bearer XXX' \
+            -H "Content-type: application/json" \
+  'https://qstash.upstash.io/v2/enqueue/my-queue/https://example.com' -d {"data"}

```

Comment:
The body should be valid JSON right?  We could do `-d '{"dataKey": "dataValue"}'`